pkg/vulnsrc/debian/debian.go

```diff
@@ -555,21 +569,25 @@ func (vs VulnSrc) parseDistributions(rootDir string) error {
 }
 
 func (vs VulnSrc) parseSources(dir string) error {
+	eb := oops.With("dir", dir)
 	for code := range vs.distributions {
 		codePath := filepath.Join(dir, code)
 		if ok, _ := utils.Exists(codePath); !ok {
 			continue
 		}
+		eb = eb.With("code_name", code)
 
 		vs.logger.Info("Parsing sources...", log.String("code", code))
 		err := utils.FileWalk(codePath, func(r io.Reader, path string) error {
+			eb := eb.With("file_path", path)
```

Comment:
same

Reply:
Regarding, https://github.com/aquasecurity/trivy-db/pull/488#discussion_r1930120127, `dir` and `path` are defined apart.
And `dir` (or `codePath`) is probably helpful for enriching errors from `utils.FileWalk`.
For example, when there is no permission on `codePath`, the context `file_path` will not be added to this error.
https://github.com/knqyf263/trivy-db/blob/b7b971c8ac657b463e70af06241a678d10bb7c7d/pkg/vulnsrc/debian/debian.go#L623

I feel like it's challenging to judge when the context should be added. That's why I took an approach of adding contexts as much as possible.